### ISSUE TITLE
Skip link does not meet contrast guidelines

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#9515]
### Issue Summary

The skip to content link, which is only available when using the keyboard to navigate the page, does not meet the contrast guidelines.

https://colourcontrast.cc/1c8767/ffffff

* `background: var(--w-color-positive-100);` = `#1c8767`
* `.button { color: var(--w-color-white)...` = `#ffffff`

![Screen Shot 2022-10-27 at 8 03 22 pm](https://user-images.githubusercontent.com/1396140/198255772-6e8c2c2a-63c8-492d-9ae8-84b79e71ec36.png)

### Steps to Reproduce

1. Open any Wagtail admin interface
2. Using a keyboard press `tab`
3. Observe that the Skip to main content button becomes visible
4. Expected: The colours on this button should meet contrast requirements
5. Actual: The white and light green are too close in contrast

#### Screenshots
![Screen Shot 2022-10-27 at 7 47 36 pm](https://user-images.githubusercontent.com/1396140/198257581-0f209f25-5303-4bbb-adcd-ea654655cf79.png)

![Screen Shot 2022-10-27 at 7 47 11 pm](https://user-images.githubusercontent.com/1396140/198257623-bea8e46a-e93a-4fcd-a78c-19db1f105bc3.png)

### Technical details

-   Wagtail version: 4.2 (in development)
-   Browser version: Firefox 105

### Recommended solution

The SCSS is in the file `client/scss/components/_skiplink.scss`

* Idea 1 - Change the background to use the new secondary color `background: theme('colors.secondary.DEFAULT');`
* Idea 2 - Leave background colour as is & button text black `color: theme('colors.black.DEFAULT');`

The colour of the focus outline also must meet contrast guidelines when compared to the background (if changed).

Please ensure any PR includes screenshots of the new skip link (remember you can use the dev tools to force 'focus' state on an element) in both small an large screens and when the sidebar is open & closed. We need to consider the overall design impacts of this change.

This has been created for Outreachy contributors as a good initial issue.

**Comment by Dev-Yusuf** [comment]
I want  to investigate this @lb-

**Comment by lb-** [comment]
Sounds good @Dev-Yusuf  - first check you can reproduce the issue and that you are able to confirm the colours do not meet the accessibility contrast guidelines.  Here are some additional tools https://css-tricks.com/color-contrast-accessibility-tools/

After that we can remove the unconfirmed label and you can start working on ideas for a fix or a PR.

**Comment by Dev-Yusuf** [comment]
Ok... Thank you.
